packages/styles/src/mixins/_mixins.scss

```diff
@@ -757,3 +757,29 @@
     }
   }
 }
+
+@mixin fd-is-compact() {
+  @at-root {
+    [class*="--compact"] & {
+      &:not([class*="--cozy"]):not([class*="--condensed"]) {
+        @content;
+      }
+    }
+
+    &[class*="--compact"] {
+      @content;
+    }
+  }
+}
+
+@mixin fd-is-condensed() {
+  @at-root {
+    [class*="--condensed"] &,
+    &[class*="--condensed"] {
+      :not([class*="--cozy"]),
+      :not([class*="--compact"]) {
+        @content;
+      }
+    }
+  }
+}
```

Comment:
mixin for cozy state? I don't remember what we decided on this? Do we avoid it because the default state is cozy?

Reply:
@InnaAtanasova I think we should avoid it. `--cozy` should be used only in one case: if parent element is `--compact` or `--condensed`, and inner element should be `--cozy`

Reply:
yes, that's exactly the case I have in mind, because in the past one of the reasons we avoided setting the `compact` modifier on the parent is that it prevents the user from mixing compact and cozy nested elements. So for example, if you set the parent to be in `compact` mode all children "inherit" it and there's no way to set a child element to `cozy` as it's the default state and doesn't have a modifier.